src/py/flwr/server/superlink/fleet/grpc_rere/fleet_servicer.py

```diff
@@ -51,27 +51,29 @@ def CreateNode(
         self, request: CreateNodeRequest, context: grpc.ServicerContext
     ) -> CreateNodeResponse:
         """."""
-        log(INFO, "FleetServicer.CreateNode")
+        log(INFO, "FleetServicer: CreateNode %s", request)
         response = message_handler.create_node(
             request=request,
             state=self.state_factory.state(),
         )
         log(INFO, "FleetServicer: Created node_id=%s", response.node.node_id)
+        log(DEBUG, "Created response: %s", response)
```

Comment:
It might help to always add the gRPC method that creates the log:
```suggestion
        log(DEBUG, "[Fleet.CreateNode] Created response: %s", response)
```